server/middleware/validation.js

```diff
@@ -280,6 +281,15 @@ const validators = {
             }
         }
         return message || validationErrors.oneOf();
+    },
+    isValidWithinNextThirtyDays({ label, value, message }) {
```

Comment:
Wonder if there's a way we can make this a generic isValidWithinDays validator where we can pass in the number of days?

I'm thinking if on the field we can set a prop on the field and access that in the validation middleware to pass through into the validator.

Also requires tests adding in `hocs-frontend/server/middleware/__tests__/validation.spec.js`

Reply:
@robertdeniszczyc2 Good idea, I've swapped it to a generic version, see what you think. Also added tests.